### DISCUSSION TITLE
Kit start

### DISCUSSION THREAD

**Discussion by FeudeyTF** [Pryaxis/TShock#2607] [Q&A]
Is there an application for the kit start?

**Comment by hakusaro** [comment]
Can you rephrase your question? I don’t quite understand.

If English isn’t your first language and you used a translator, can you post in your original language or use https://www.deepl.com/translator instead?

**Reply by FeudeyTF** [reply]
Where can I find a plugin so that a player with a privilege can get a starter kit
